components/cli/pkg/commands/project/init.go

```diff
@@ -108,27 +163,170 @@ func RunInit(cli cli.Cli, projectName string) error {
 					},
 				},
 			},
-		}); err != nil {
-			return fmt.Errorf("error occurred while initializing the project, %v", err)
+		})
+		if err != nil {
+			util.ExitWithErrorMessage("Error occurred while initializing the project", err)
 		}
 	}
-	projectDir := filepath.Join(cli.FileSystem().CurrentDir(), projectName)
-	if err := util.CreateDir(projectDir); err != nil {
-		return fmt.Errorf("failed to initialize project, %v", err)
-	}
-	writer, err := os.Create(filepath.Join(projectDir, projectName+".bal"))
+	currentDir, err := os.Getwd()
 	if err != nil {
-		return fmt.Errorf("error in creating Ballerina File, %v", err)
-	}
-	balW := bufio.NewWriter(writer)
-	if _, err = balW.WriteString(CellTemplate); err != nil {
-		return fmt.Errorf("failed to create writer for cell file, %v", err)
-	}
-	if _ = balW.Flush(); err != nil {
-		return fmt.Errorf("failed to cleanup writer for cell file, %v", err)
+		util.ExitWithErrorMessage("Error in getting current directory location", err)
```

Comment:
Return the error instead of exiting, in all places.

Reply:
Fixed